SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDevAuthTokenGenerator.cpp

```diff
@@ -46,10 +47,10 @@ void FSpatialGDKDevAuthTokenGenerator::DoGenerateDevAuthTokenTasks()
 		}
 		else
 		{
-			UE_LOG(LogSpatialGDKDevAuthTokenGenerator, Error, TEXT("Failed to generate a development authentication token."));
-			AsyncTask(ENamedThreads::GameThread, [this]()
+			UE_LOG(LogSpatialGDKDevAuthTokenGenerator, Error, TEXT("Failed to generate a Development Authentication Token:%s"), *ErrorMessage);
+			AsyncTask(ENamedThreads::GameThread, [this,&ErrorMessage]()
```

Comment:
I don't think we should add the error message to the notification popup. It doesn't have much space so it would look confusing. Having the error in the output log should be sufficient.
```suggestion
			AsyncTask(ENamedThreads::GameThread, [this]()
```